src/Sylius/Bundle/ApiBundle/ApiPlatform/Bridge/Symfony/Routing/RouteNameResolver.php

```diff
@@ -72,8 +72,8 @@ private function isSameSubresource(array $context, array $currentContext): bool
         $subresources = array_keys($context['subresource_resources']);
         $currentSubresources = [];
 
-        foreach ($currentContext['identifiers'] as $identifierContext) {
-            $currentSubresources[] = $identifierContext[1];
+        foreach ($currentContext['identifiers'] as [$class]) {
```

Comment:
First time seeing this syntax, what does it do? 😨

Reply:
This solution is the same like new approach in `RouteNameResolver` in api-platform 2.6, If I good understand, this syntax means that if element of array is not array it is null:
<img width="589" alt="Screenshot 2021-04-12 at 10 04 46" src="https://user-images.githubusercontent.com/39232096/114361567-8c773180-9b76-11eb-87a5-66a8bfbb3c08.png">